allennlp_server/commands/configure.py

```diff
@@ -0,0 +1,54 @@
+"""
+The ``configure`` subcommand launches a webapp that helps you generate an AllenNLP configuration file.
+
+    $ allennlp configure --help
+    usage: allennlp configure [--help] [-h HOST] [-p PORT] [--include-package INCLUDE_PACKAGE]
+
+    Run a config explorer for allennlp.
+
+    optional arguments:
+      --help                show this help message and exit
+      -h HOST, --host HOST  interface to serve the wizard on (default = 127.0.0.1)
```

Comment:
Using `-h` collides with `--help`, but I saw this is what `flask run` uses, so it could be expected by users. I don't have a strong opinion on this though.